src/renderers/shared/utils/CallbackQueue.js

```diff
@@ -101,6 +102,4 @@ Object.assign(CallbackQueue.prototype, {
 
 });
 
-PooledClass.addPoolingTo(CallbackQueue);
-
-module.exports = CallbackQueue;
+module.exports = PooledClass.addPoolingTo(CallbackQueue);
```

Comment:
I'm afraid that doing

```js
CallbackQueue = PooledClass.addPoolingTo(CallbackQueue);
```

is going to confuse flow in the file (I haven't tried).

If creating a new variable, I'm not sure how to call it, maybe

```
var PooledCallbackQueue = PooledClass.addPoolingTo(CallbackQueue);
module.exports = PooledCallbackQueue;
```

which is also inconsistent with the `ThingWithSameNameAsFile` rule.

There's precedent with this in the Relay world where you first declare your React component and then do

```js
module.exports = Relay.createContainer(ReactComponent, { ... graphql query ... });
```

People are also doing this with Redux

```js
module.exports = connect(...
```

I'm happy to change it to something else :)